lib/que/locker.rb

```diff
@@ -133,40 +146,31 @@ def exists?(job)
       end
     end
 
-    def lock_job_query
-      if @secondary_queues.any?
-        Que.execute(
-          :queue_permissive_lock_job,
```

Comment:
Can probably ✂️ the query also

Reply:
yeah, was going to before I turned this PR out of draft